Opserver/Controllers/GraphController.Spark.cs

```diff
@@ -92,18 +93,18 @@ public ActionResult SQLCPUSpark(string node)
                  divisor = max/50,
                  range = (nowEpoch - startEpoch)/width;
 
-            var sb = StringBuilderCache.Get().AppendFormat(@"<svg version=""1.1"" baseProfile=""full"" width=""{0}"" height=""{1}"" xmlns=""http://www.w3.org/2000/svg"" preserveAspectRatio=""none"">
+            var sb = StringBuilderCache.Get().AppendFormat(CultureInfo.InvariantCulture, @"<svg version=""1.1"" baseProfile=""full"" width=""{0}"" height=""{1}"" xmlns=""http://www.w3.org/2000/svg"" preserveAspectRatio=""none"">
   <line x1=""0"" y1=""{1}"" x2=""{0}"" y2=""{1}"" stroke=""{3}"" stroke-width=""1"" />
   <g fill=""{2}"" stroke=""none"">
-    <path d=""M0 50 L", width.ToString(), height.ToString(), Color, AxisColor);
+    <path d=""M0 50 L", width, height, Color, AxisColor);
             foreach (var p in points)
             {
-                sb.Append((p.DateEpoch - startEpoch) / range).Append(" ")
-                  .Append((height - getVal(p) / divisor).ToString("n1")).Append(" ");
+                sb.Append(((p.DateEpoch - startEpoch) / range).ToString(CultureInfo.InvariantCulture)).Append(" ")
+                  .Append((height - getVal(p) / divisor).ToString("n1", CultureInfo.InvariantCulture)).Append(" ");
             }
-            sb.AppendFormat(@"{0} {1} z""/>
+            sb.AppendFormat(CultureInfo.InvariantCulture, @"{0} {1} z""/>
    </g>
-</svg>", width.ToString(), height.ToString());
```

Comment:
For the integer case specifically, the only thing that can change for integers (the only thing involves in the `.AppendFormat()` cases) is the negative sign, which isn't in play. So we can leave those `.ToString()` to eliminate boxing by their original calls.